apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.CommonTesting/SpannerTestDatabaseBase.cs

```diff
@@ -113,15 +110,11 @@ protected SpannerTestDatabaseBase(string projectId, EmulatorDetection emulatorDe
         // In production, we expect the instance to already exist.
         MaybeCreateInstanceOnEmulator(projectId);
 
-        if (Fresh)
-        {
-            CreateDatabase();
-            Logger.DefaultLogger.Debug($"Created database {SpannerDatabase}.");
-        }
-        else
-        {
-            Logger.DefaultLogger.Debug($"Using existing database {SpannerDatabase}");
-        }
+        // Always attempt to create the DB ignoring AlreadyExists error and mark fresh if successfull.
+        CreateDatabase();
+        Fresh = true;
```

Comment:
I haven't looked in detail, but it appears that this will now always be true, irrespective of if database is created or not. Am I missing something?

Reply:
And what will happen if database already exists? Before Fresh is true, exception will be thrown, based on current implementation of GSQL  `CreateDatabase`

Reply:
Possibly, change `protected abstract void CreateDatabase();` to return bool. And return `True` if fresh and `False` if it already exists.

Reply:
Thanks for feedback. Rectified the code.